syntax = "proto3";
import "book.proto";

service InventoryService {
  rpc CreateBook(CreateBookRequest) returns (Response) {}

  rpc GetBook(GetBookRequest) returns (Book) {}
}

message CreateBookRequest {
  Book book = 1;
}

message Response {
  int32 code = 1;
}

message GetBookRequest {
  string isbn = 1;
}